beacon-chain/core/blocks/block_operations_test.go

```diff
@@ -1825,3 +1825,296 @@ func TestProcessValidatorExits_AppliesCorrectStatus(t *testing.T) {
 			helpers.DelayedActivationExitEpoch(state.Slot/params.BeaconConfig().SlotsPerEpoch), newRegistry[0].ExitEpoch)
 	}
 }
+
+func TestProcessBlockHeader_OK(t *testing.T) {
+	t.Skip()
+	// TODO(#2307) unskip after bls.Verify is finished
```

Comment:
why is this skipped ?, bls.Verify is commented out

Reply:
removed